docs/reference/search-api.md

```diff
@@ -69,3 +70,40 @@ Search for documents matching a query in the given index `<index name>`.
 | **hits**             | Results of the query           | `[hit]` |
 | **numHits**         | Total number of matches        |  `number`  |
 | **numMicrosecs**    | Processing time of the query   |  `number`  |
+
+
+
+### Export search from an index
+
+```
+GET api/v1/indexes/<index name>/search/stream
+```
+
+Export documents matching a search query in the given index `<index name>`, in a specified output format.
+
+The output format can be one of the following:
+ -  [Comma Separated Values](https://datatracker.ietf.org/doc/html/rfc4180)
+ -  [ClickHouse Row Binary](https://clickhouse.tech/docs/en/interfaces/formats/#rowbinary)
+
+#### Path variable
+
+| Variable      | Description   |
+| ------------- | ------------- |
+| **index name** | The index name |
+
+
+#### Get parameters
+
+| Variable | Type | Description | Default value |
+|----------|------|-------------|---------------|
+| **query** | `String` | Query text. See the [query language doc](query-language.md) (mandatory) | |
+| **fastField** | `String` | Name of fast field to retrieve from documents (mandatory)| |
+| **searchFields** | `[String]` | If set, specify the set of fields the search will performed on | |
+| **startTimestamp** | `i64` | If set, restrict search to documents with a `timestamp >= start_timestamp` | |
+| **endTimestamp** | `i64` | If set, restrict search to documents with a `timestamp < end_timestamp`` | |
+| **outputFormat** | `String` | Response output format. `csv` or `clickHouseRowBinary`  | `csv` | |
+
+
+### Response
+
+The response is a list of fast field values. The formatting is based on the specified output format. 
```

Comment:
We can be more accurate. We return all the field values from documents that match the query. 
It works only if the field is marked as fast in the index config.